base/logging/logger.go

```diff
@@ -27,7 +27,7 @@ func ParseLoglevel(level string) (logrus.Level, error) {
 	case "FATAL":
 		return logrus.FatalLevel, nil
 	default:
-		return logrus.InfoLevel, errors.New("Invalid loglevel given")
+		return logrus.InfoLevel, errors.New("invalid loglevel given")
```

Comment:
I'm curious, why this change?

Reply:
Error string should not be capitalized or end with punctuation mark 
https://github.com/golang/go/wiki/CodeReviewComments#error-strings